modules/bullet/space_bullet.cpp

```diff
@@ -438,16 +440,27 @@ real_t SpaceBullet::get_param(PhysicsServer3D::SpaceParameter p_param) {
 }
 
 void SpaceBullet::add_area(AreaBullet *p_area) {
+#ifdef TOOLS_ENABLED
+	// This can't never happen at this piont.
+	CRASH_COND(p_area->is_in_world);
+#endif
```

Comment:
I'm not convinced that this can never happen. So, in case it does, instead of silently crashing with `CRASH_COND`, it would be better to use `ERR_FAIL_COND_MSG` with an explanation to assist future debugging and let the IDE continue running.

Reply:
This can't never happen, there is no way for the user to trigger it. It's there as notification bell for the future developers; if someone change the code and for a mistakes any body is added twice the condition is triggered and the debugger will tell him why.

By the way, bullet would crash anyway now that `btAssert` is again used.